diff --git a/node_modules/expo/scripts/react-native-xcode.sh b/node_modules/expo/scripts/react-native-xcode.sh
index e6ae4af..e960353 100644
--- a/node_modules/expo/scripts/react-native-xcode.sh
+++ b/node_modules/expo/scripts/react-native-xcode.sh
@@ -25,7 +25,7 @@ if [[ ! "$SKIP_BUNDLING_METRO_IP" && "$CONFIGURATION" = *Debug* && ! "$PLATFORM_
     IP=$(ifconfig | grep 'inet ' | grep -v ' 127.' | grep -v ' 169.254.' |cut -d\   -f2  | awk 'NR==1{print $1}')
   fi
 
-  echo "$IP" > "$DEST/ip.txt"
+  echo "$IP" > "/tmp/ip.txt"
 fi
 
 if [[ "$SKIP_BUNDLING" ]]; then
